db/helpers/common.js

```diff
@@ -188,7 +189,18 @@ function publishChange(inst, event, changedKeys, ignoreAttributes) {
   }
 
   if (obj[event]) {
-    pub.publish(channelName, JSON.stringify(obj));
+    const objectAsString = JSON.stringify(obj);
+    zlib.deflate(objectAsString, (err, zippedValue) => {
```

Comment:
Put the zlib stringify/deflate/toString and unzip/parse functions into a util file which can be called from here, from redisPublisher and from redisSubscriber, then add some tests for the two util functions